app/views/posts/index.html.erb

```diff
@@ -1,3 +1,12 @@
-<h1>Posts List</h1>
-
-<p>Here is a list of all posts for a given user based on the given user_id</p>
\ No newline at end of file
+<section class="users">
+<% if @posts.exists? %>
+      <%= render partial: "user_posts", locals: { post: @posts.first.user } %>
+    <% @posts.each do |post|  %>
+    <div>
+        <%= render partial: "post_cards", locals: { post: post } %> 
+    </div>
+    <% end %>
+<% else %>
+    <p>No posts found for this user</p>
+<% end %>
+</section>
```

Comment:
- [ ] Please Uses `post.recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb` page to match the wireframe design given.

![image](https://user-images.githubusercontent.com/68796232/180611038-f91a8544-9e1b-4898-9e6c-16069f8fa7a3.png)

Reply:
Hello @Omar-Muhamad , Thanks for your feedback but as you see, the method is implemented into this file. 

![Screenshot 2022-07-23 at 17 30 36](https://user-images.githubusercontent.com/86868013/180611751-02956462-3b95-4fec-bedd-58bfdf9ad7b5.png)